shared/README.md

```diff
@@ -45,7 +45,7 @@ The key things to keep in mind are:
   - Receive asynchronous events from the **main events channel**
 - The **Persistence** module is the only module that communicates with the local database
 - The **P2P** module is the only one that communicates with the outside world
-
+- **Clock** is a drop-in replacement for some of the features offered by the `time` package, it acts as an injectable clock implementation used to provide time manipulation while testing.
```

Comment:
make sure to update the changelog as well

Reply:
Addressed in https://github.com/pokt-network/pocket/pull/198/commits/09f1914ba2e85be46dbac330f48e48b403fe93db